moneriote.py

```diff
@@ -1,303 +1,502 @@
+#!/usr/bin/python3.5
```

Comment:
My python 3.6.5 can't run with this. But I don't know the ideal way to handle this.